sql/updates/world/2015_10_01_14938_world.sql

```diff
@@ -0,0 +1,457 @@
+(@CRITERIA_VEZAX,1,35051),
+(@CRITERIA_ALGALON,1,35052);
+
+-- Correcting Paletress' speed values on heroic mode
+UPDATE `creature_template` SET `speed_walk`=2,`speed_run`=1.42857 WHERE `entry`=35517;
+
+-- Correcting Paletress' mechanic immunities
+-- she is interruptable
+UPDATE `creature_template` SET `mechanic_immune_mask`=617299839 WHERE `entry` IN (34928,35517);
+
+-- Paletress' Memory adds are missing mechanic immunities
+UPDATE `creature_template` SET `mechanic_immune_mask`=650854271 WHERE `entry`=34942 OR `entry` BETWEEN 35028 AND 35052 OR `entry` BETWEEN 35519 AND 35525 OR `entry` BETWEEN 35527 AND 35544;
+
+-- Correcting Eadric' mechanic immunities
+UPDATE `creature_template` SET `mechanic_immune_mask`=650854271 WHERE `entry` IN (35119,35518);
+
+-- Eadric' Vengeance spell should have internal cooldown and procChance
+-- Proc spell should properly be removed on successful melee crit
+DELETE FROM `spell_proc_event` WHERE `entry` IN (66865,66889);
+INSERT INTO `spell_proc_event` (`entry`,`procFlags`,`procEx`,`CustomChance`,`Cooldown`) VALUES 
+(66865,4,0,20,10),
+(66889,4,2,0,0);
+-- Also the proc spell of Vengeance should not remove original aura
+DELETE FROM `spell_linked_spell` WHERE `spell_trigger`=66889 AND `spell_effect`=-66865;
+
+-- Player casted Hammer of the Righteous to Eadric should remove the dummy aura from player on cast
+DELETE FROM `spell_linked_spell` WHERE `spell_trigger`=66905 AND `spell_effect`=-66904;
+INSERT INTO `spell_linked_spell` VALUES (66905,-66904,0,'Eadric - Remove dummy aura from player on cast');
+
+--
+-- The Black Knight
+--
+
+-- Variables
+SET @KNIGHT := 35451;
+SET @KNIGHT_H := 35490;
+SET @KNIGHT_VEHICLE := 35491;
+SET @RISEN_CHAMP := 35590;
+SET @RISEN_CHAMP_H := 35717;
+
+-- The Black Knight's vehicle must have flight inhabit type
+UPDATE `creature_template` SET `InhabitType`=7 WHERE `entry`=@KNIGHT_VEHICLE;
+
+-- Risen Champion is missing scriptname
+UPDATE `creature_template` SET `ScriptName`='npc_risen_ghoul' WHERE `entry`=@RISEN_CHAMP;
+
+-- Risen Champions (not Risen Herald) and The Black Knight should award reputation on kill
+DELETE FROM `creature_onkill_reputation` WHERE `creature_id` IN (@RISEN_CHAMP,@RISEN_CHAMP_H,@KNIGHT,@KNIGHT_H);
+INSERT INTO `creature_onkill_reputation` VALUES
+(@RISEN_CHAMP,1037,1052,7,0,5,7,0,5,1),
+(@RISEN_CHAMP_H,1037,1052,7,0,10,7,0,10,1),
+(@KNIGHT,1037,1052,7,0,195,7,0,195,1),
+(@KNIGHT_H,1037,1052,7,0,390,7,0,390,1);
+
+-- Correcting The Black Knight's mechanic immunities
+UPDATE `creature_template` SET `mechanic_immune_mask`=650854271 WHERE `entry` IN (@KNIGHT,@KNIGHT_H);
+
+-- The Black Knight should have money drop
+UPDATE `creature_template` SET `mingold`=4921,`maxgold`=10294 WHERE `entry`=@KNIGHT;
+UPDATE `creature_template` SET `mingold`=14691,`maxgold`=21952 WHERE `entry`=@KNIGHT_H;
+
+-- Achievement criteria script for achievement id 3804
+DELETE FROM `achievement_criteria_data` WHERE `type` IN (11, 18) `criteria_id`=11789;
```

Comment:
Missing `AND` in this line seems to be the only thing Travis complains about currently. Should be:
```sql
DELETE FROM `achievement_criteria_data` WHERE `type` IN (11, 18) AND `criteria_id`=11789;
```